openedx/core/djangoapps/programs/views.py

```diff
@@ -0,0 +1,83 @@
+"""
+Platform support for Programs.
+
+This package is a thin wrapper around interactions with the Programs service,
+supporting learner- and author-facing features involving that service
+if and only if the service is deployed in the Open edX installation.
+
+To ensure maximum separation of concerns, and a minimum of interdependence's,
+this package should be kept small, thin, and stateless.
+"""
```

Comment:
this docstring describes the package itself, so it should be in `__init__` or a README, not in the views module.